### PR TITLE
Add tests for Altimeter

### PR DESCRIPTION
**Pull request by sishamilton**
The goal of this pull request is to add tests for the Webots altimeter ROS API, which is proposed in PR [#3029](https://github.com/cyberbotics/webots/pull/3029) in the main Webots repository.

This pull request will add an altimeterCallback() function as well as an altimeter methods test to /src/complete_test.cpp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,18 +12,34 @@ ROS tutorial for Webots:
...
+</a></br>
+
+Supported by OpenDR - Open Deep Learning Toolkit for Robotics.  
+More information: <a href="https://opendr.eu/">opendr.eu</a>
+
+<img src="https://opendr.csd.auth.gr/wp-content/uploads/2019/12/Flag_of_Europe-300x200.png"
+     alt="eu_flag" height="45" align="left" >  
+
+This project has received funding from the European Union’s Horizon 2020  
+research and innovation programme under grant agreement no. 871449.
```
Comment (lukicdarkoo) [inline README.md:45]: This file should not be changed

File: src/complete_test.cpp [hunk 2]
```diff
@@ -1025,6 +1030,58 @@ int main(int argc, char **argv) {
...
 
+  ////////////////////////////
+  // ALTIMETER METHODS TEST //
+  ////////////////////////////
+
+  ros::ServiceClient set_altimeter_client;
+  webots::set_int altimeter_srv;
+  ros::subscriber sub_altimeter_32;
+
+  set_altimeter_client = n.serviceClinet<webots_ros::set_int>(model_name + "/altimeter/enable");
```
Comment (lukicdarkoo) [inline src/complete_test.cpp:1041]: Did you run the test? It doesn't look like it would pass as it is added in the robot

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
The structure looks good, but make sure the test is failing:
```
roslaunch webots_ros complete_test.launch
```

File: README.md (hunk 1 above)
Comment (sishamilton) [reply README.md:45]: As far as I can tell these changes were not introduced by me. I believe they came from this commit [4a52aa2](https://github.com/cyberbotics/webots_ros/pull/63/commits/4a52aa2b89955ae73dcc762e59c8b5e5fb0eca03). Should I still remove these changes?

**Comment by sishamilton** [comment]
complete_test is now running succesfully. I believe that this PR is ready to be merged pending the resolution of @lukicdarkoo's comment on the changed readme.

File: README.md (hunk 1 above)
Comment (lukicdarkoo) [reply README.md:45]: You probably forked from an old version, merging the develop branch into this one should resolve the issue

Comment (sishamilton) [reply README.md]: Yes the problem was that I forked the master branch rather than the develop. This has been resolved.

**Comment by sishamilton** [comment]
This PR should be ready to merge.

**Comment by lukicdarkoo** [comment]
@sishamilton Thank you, but make sure this is merged first:
https://github.com/cyberbotics/webots/pull/3029
Then we can review this one

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Pull request merged**